registry/handlers/manifests.go

```diff
@@ -65,17 +66,58 @@ type manifestHandler struct {
 
 // GetManifest fetches the image manifest from the storage backend, if it exists.
 func (imh *manifestHandler) GetManifest(w http.ResponseWriter, r *http.Request) {
-	ctxu.GetLogger(imh).Debug("GetImageManifest")
+	ctxu.GetLogger(imh).Debug("GetManifest")
```

Comment:
I think what happened here is that the master branch renamed GetImageManifest to GetManifest (but forgot to update this log message), and when you rebased, you accidentally duplicated things so that there is now both a GetManifest and an (unused) GetImageManifest in this file that need to be combined into a single version.